src/ONYXKEYS.ts

```diff
@@ -376,6 +391,10 @@ const ONYXKEYS = {
 
         // Search Page related
         SNAPSHOT: 'snapshot_',
+
+        // Shared NVPs
+        /** Collection of objects where each objects represents a workspace’s owner which is past due billing AND the user is a member of. */
```

Comment:
Would this sound better?
```suggestion
        /** Collection of objects where each object represents a workspace’s non-owner(normal users and admins) who is member of a workspace where the owner is past due billing.  */
```

Reply:
@rojiphil That is not true, each object represents the workspace owner where the user is member of. I've changed to `Collection of objects where each object represents the owner of the workspace that is past due billing AND the user is a member of.` Wdyt?

Reply:
Ah! You are right. It makes sense too.